package.json

```diff
@@ -45,7 +46,7 @@
     "@lit-protocol/misc": "^7.0.8",
     "@lit-protocol/pkp-ethers": "^7.0.8",
     "@lit-protocol/types": "^7.0.8",
-    "@lit-protocol/vincent-sdk": "0.1.1-1",
+    "@lit-protocol/vincent-sdk": "0.0.2",
```

Comment:
Bumped to 0.0.2 and updated the dashboard to use the new `jwt` API.

Reply:
It is, a bit -- I should've commented on why they are different -- this PR's SDK changes _become_ 0.0.4, but as part of this same PR I updated the dashboard to use the new APIs that were published in 0.0.2.  The changes in 0.0.4 aren't applicable to the dashboard (it adds express middleware) so I didn't explicitly update it...

Next time I'll make a separate PR to update the dashboard, or I'll note it explicitly :+1: thx for the heads up